### ISSUE TITLE
[BUG] VS Code extension not working

### ISSUE THREAD

**Issue by MarcusFelling** [microsoft/playwright-vscode#62]
**Context:**
- Playwright Version: 1.15.2
- Extension version: v0.0.7
![image](https://user-images.githubusercontent.com/6855361/137360615-967c9628-a3fc-4f59-b78f-01158f9c0c8c.png)

**Describe the bug**
Test explorer does not appear when viewing test files with extension `*.spec.ts`

**Output window log**
![image](https://user-images.githubusercontent.com/6855361/137361852-f8f91117-8e72-4c63-8e16-b30f0c0d56f0.png)

![image](https://user-images.githubusercontent.com/6855361/137361746-c0fe5871-d734-4623-a7ee-8680957deb39.png)

**Comment by MarcusFelling** [comment]
@F3n67u - thanks, that does seem to be the issue. Opening the tests at root gets the extension working.

**Comment by mxschmitt** [comment]
Yeah as already pointed out currently the extension only works if you have `@playwright/test` in the root package.json. This is mostly the case for normal single package.json users or when using e.g. Yarn/NPM workspaces.

In your case since you have a root package.json and one in a subfolder, you can make it work by adjusting the config a bit:

```json
{
    "playwright.cliPath": "./e2eTests/node_modules/@playwright/test/lib/cli/cli.js",
    "playwright.configs": [
        "e2eTests/playwright.config.ts"
    ],
    "playwright.env": {
        "SITE_URL": "http://connect.schmitt.mx"
    }
}
```

I see potential in a separate option called `playwright.cwd` so you don't need to specify `playwright.configs` and `playwright.cliPath` manually.
